docs/newsletter.rst

```diff
@@ -42,3 +42,222 @@ If you have any questions or want to contribute to the newsletter, go ahead and
   </div>
   <script type='text/javascript' src='//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js'></script><script type='text/javascript'>(function($) {window.fnames = new Array(); window.ftypes = new Array();fnames[0]='EMAIL';ftypes[0]='email';}(jQuery));var $mcj = jQuery.noConflict(true);</script>
   <!--End mc_embed_signup-->
+
+Index of Newsletter Topics
+--------------------------
+
+All the topics we've covered in the newsletter. In no particular order,
+and some topics are repeated if they fit into more than one category. Enjoy!
+
+Collaboration
+~~~~~~~~~~~~~
+
+- `Helping project managers understand docs </blog/newsletter-june-2018/#helping-your-project-managers-understand-docs>`_
+- `Common misconceptions about docs </blog/newsletter-august-2018/#common-misconceptions-about-documentation>`_
+- `Conflicts about what should be in the docs </blog/newsletter-july-2017/#who-s-running-this-content>`_
+- `Writing good bug reports </blog/newsletter-april-2017/#the-art-of-the-bug-report>`_
+
+Helping others to write
+^^^^^^^^^^^^^^^^^^^^^^^
+
+- `Promoting plain language </blog/newsletter-november-2018/#promoting-plain-language>`_
+- `Teaching peers writing </blog/newsletter-may-2017/#peer-to-peer-teaching>`_
+- `Giving feedback </blog/newsletter-february-2018/#it-s-not-personal-it-s-feedback>`_
+
+Workflows and Agile
+^^^^^^^^^^^^^^^^^^^
+
+- `Making docs maintainable </blog/newsletter-august-2017/#making-docs-maintainable>`_
+- `Task management for docs teams </blog/newsletter-october-2018/#task-management-tools-for-docs-teams>`_
+- `Agile workflows for docs </blog/newsletter-october-2018/#show-us-your-workflows>`_
+- `Working on Agile teams </blog/newsletter-may-2017/#the-challenges-of-documentation-in-an-agile-environment>`_
+- `Changing docs workflows </blog/newsletter-february-2017/#best-practices-for-changing-your-docs-workflow>`_
+- `Automating routine tasks </blog/newsletter-november-2017/#to-automate-or-not-to-automate>`_
+
+Style guides
+^^^^^^^^^^^^
+
+- `Enforcing UI style guides </blog/newsletter-june-2017/#the-enforcer-ui-style-guides-edition>`_
+- `Style guides we like </blog/newsletter-november-2016/#a-quick-guide-to-style-guides>`_
+- `One style guide for all types of writing? </blog/newsletter-september-2016/#one-style-guide-or-two>`_
+
+Encouraging contributions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+See also: helping others to write
```

Comment:
```suggestion
See also :doc:`helping-others-to-write`
```

Reply:
Not 100% sure about this syntax for an in page anchor

Reply:
This is worth doing, but not going to block merge on it 👍